### ISSUE TITLE
Upgrade Jetty to the latest version

### ISSUE THREAD

**Issue by jo-elimuai** [elimu-ai/webapp#599]
Upgrade to Jetty 9

See https://www.eclipse.org/jetty/

- [x] Decide which Jetty version to upgrade to
- [x] Upgrade Maven plugin
- [x] Configure the same Jetty version on the test server (test.elimu.ai)
- [x] Configure the same Jetty version on the prod server elimu.ai)

**Comment by jo-elimuai** [comment]
@aschworer @syntrydy Will it be possible to run Jetty 9 in a Docker container? - https://github.com/elimu-ai/webapp/issues/598

**Comment by syntrydy** [comment]
yes, we can run jetty on docker. But to make that works with our application, we have to do some scripting.

**Comment by aschworer** [comment]
> Decide which Jetty version to upgrade to
I would say to use latest (9.4.7.v20170914) unless any problem pops up

**Comment by aschworer** [comment]
@jo-elimuai 
I configured Jetty 9 on TEST, and made the webapp run on it,

The only thing required to make it work is upgrading `jetty-jspc-maven-plugin` version to the jetty 9 version (9.4.7.v20170914), as the pre-compiled JSPs have to comply with upgraded Jetty.

Another option is to turn-off JSP pre-compilation for a while. In this case no change will be required to run on Jetty 9.

I had to stop for now, as the server run out of space and Jetty wouldn't start anymore.
It still has old configuration and JETTY_HOME, so `/etc/init.d/jetty start` will work as soon as some space is available.
